less/paperhive/intro.less

```diff
@@ -20,6 +20,18 @@
   }
 }
 
+.ph-padding-20 {
+  padding: 20px;
+}
+
+.ph-margin-bottom-10 {
+  margin-bottom: 10px;
+}
+
+.ph-font-size-small {
+  font-size: small;
+}
+
```

Comment:
We should try to use the util classes from https://github.com/paperhive/paperhive-frontend/blob/master/less/paperhive/utils.less#L19 wherever possible...